### PR TITLE
Increase input field width for varchar(255)

### LINKED ISSUE #13627: Increase input field width for varchar(255)
The input field width (on Insert Row page) is currently 40 chars, could this be increased?
Perhaps the value fields could use all remaining width in the table.

### PR DESCRIPTION
**Pull request by shucon**
Fixes: https://github.com/phpmyadmin/phpmyadmin/issues/13627

Signed-off-by: Saksham Gupta <shucon01@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by nulll-pointer** [comment]
I think relative width should be used.

**Comment by shucon** [comment]
Hmm, Thanks for pointing it out.

File: libraries/classes/InsertEdit.php [hunk 1]
```diff
@@ -954,12 +954,18 @@ private function getTextarea(
...
             );
             $maxlength = $extracted_columnspec['spec_in_brackets'];
+            if ($maxlength == 255) {
+                // Special case for varchar(255)
+                $textAreaRows = $GLOBALS['cfg']['CharTextareaRows'] * 3;
+                $textareaCols = $GLOBALS['cfg']['CharTextareaCols'] * 2;
+            } else {
+                $textAreaRows = $GLOBALS['cfg']['CharTextareaRows'];
+                $textareaCols = $GLOBALS['cfg']['CharTextareaCols'];
+            }
```
Comment (mauriciofauth) [inline libraries/classes/InsertEdit.php]: I'm not sure if adding a special case is a good solution.

Comment (shucon) [reply libraries/classes/InsertEdit.php]: @mauriciofauth other possible solution is ` $textAreaRows = $GLOBALS['cfg']['CharTextareaRows'] * (($maxlength %100)+1);`

**Comment by mauriciofauth** [comment]
I don't know if this would be a good option, maybe changing the default values is more appropriate.

https://github.com/phpmyadmin/phpmyadmin/blob/aade3546b278cb303a8c4f7ad46b64adb1b31524/libraries/config.default.php#L2675-L2687

@phpmyadmin/developers What do you think of this?

**Comment by shucon** [comment]
Changing the default values will lead to unnecessary space allocation to text areas where it is not required, IMO.

**Comment by shucon** [comment]
@mauriciofauth this PR has gone stale, what do you think should be done?

**Comment by shucon** [comment]
Closing in favour of https://github.com/phpmyadmin/phpmyadmin/pull/14379

**Pull request closed without merging**